javascript/ecmascript-2015/promises/introducing-promises.md

```diff
@@ -24,25 +24,17 @@ links:
 ---
 ## Content
 
-A **promise** is an object that represents a value that may or may not be obtained in the future and can be passed around like any other object.
+A **promise** is an object that represents a value that may or may not be obtained in the future and can be passed around like any other value.
 
-You may have come across promises in JavaScript before either in the language itself, libraries such as jQuery (which some would argue are not really promises) and libraries such as q or rsvp.
+### Why use promises?
 
-Some of these libraries conform (and add their own additional functionality as well) to a specification called **Promise/A+**. ES6 bases its promise implementation on this specification.
+It is common in JavaScript to perform tasks that do not complete immediately – for example making an ajax call. Pages would run very slowly if we had to stop everything on a page to wait for an ajax call so JavaScript continues to execute other code.
```

Comment:
The end of the sentence might be superfluous:

> so JavaScript continues to execute other code.

Maybe say:

> It is common in JavaScript to perform tasks that do not complete immediately – for example making an ajax call. Pages would run very slowly if we had to block everything to wait for an ajax call to finish.

Reply:
done